.github/workflows/delete_doc_comment.yml

```diff
@@ -3,6 +3,9 @@ name: Delete dev documentation
 on:
   pull_request:
     types: [ closed ]
+    paths:
+      - "docs/**"
+      - "bindings/python/py_src/**"
```

Comment:
combination of `types` & `paths` should work right?

Reply:
It should yes. It's an OR here.

Reply:
oh, I was expecting AND

in this case https://github.com/huggingface/safetensors/pull/88

Reply:
Ohhh sorry, types and paths is and AND, however multiples paths is OR between each other.